src/display/annotation_layer.js

```diff
@@ -67,7 +67,16 @@ AnnotationElementFactory.prototype =
         return new TextAnnotationElement(parameters);
 
       case AnnotationType.WIDGET:
-        return new WidgetAnnotationElement(parameters);
+        switch (parameters.data.fieldType) {
+          case 'Tx':
+            return new TextWidgetAnnotationElement(parameters);
+          default:
+            warn('Unimplemented Widget annotation type: ' +
+                  parameters.data.fieldType);
+            // Fallback to default one, which does not render anything
```

Comment:
I assume it has been done with a `switch` because the intention is to add more types later, but perhaps until that happens it's better to do it like suggested above.